[2024-05-24T12:01:08.586+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-24T12:01:08.596+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: upload_from_github_to_s3.download_and_upload_to_s3 scheduled__2023-10-08T00:00:00+00:00 [queued]>
[2024-05-24T12:01:08.600+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: upload_from_github_to_s3.download_and_upload_to_s3 scheduled__2023-10-08T00:00:00+00:00 [queued]>
[2024-05-24T12:01:08.601+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-05-24T12:01:08.607+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): download_and_upload_to_s3> on 2023-10-08 00:00:00+00:00
[2024-05-24T12:01:08.610+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=679) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-24T12:01:08.610+0000] {standard_task_runner.py:63} INFO - Started process 686 to run task
[2024-05-24T12:01:08.611+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'upload_from_github_to_s3', 'download_and_upload_to_s3', 'scheduled__2023-10-08T00:00:00+00:00', '--job-id', '481', '--raw', '--subdir', 'DAGS_FOLDER/upload_from_github_to_s3_dag.py', '--cfg-path', '/tmp/tmpvnfyrblz']
[2024-05-24T12:01:08.612+0000] {standard_task_runner.py:91} INFO - Job 481: Subtask download_and_upload_to_s3
[2024-05-24T12:01:08.631+0000] {task_command.py:426} INFO - Running <TaskInstance: upload_from_github_to_s3.download_and_upload_to_s3 scheduled__2023-10-08T00:00:00+00:00 [running]> on host b2ab95bd73fd
[2024-05-24T12:01:08.670+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='upload_from_github_to_s3' AIRFLOW_CTX_TASK_ID='download_and_upload_to_s3' AIRFLOW_CTX_EXECUTION_DATE='2023-10-08T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-10-08T00:00:00+00:00'
[2024-05-24T12:01:08.670+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-24T12:01:09.132+0000] {credentials.py:1147} INFO - Found credentials in environment variables.
[2024-05-24T12:01:09.195+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-24T12:01:09.195+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-24T12:01:09.199+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=upload_from_github_to_s3, task_id=download_and_upload_to_s3, run_id=scheduled__2023-10-08T00:00:00+00:00, execution_date=20231008T000000, start_date=20240524T120108, end_date=20240524T120109
[2024-05-24T12:01:09.241+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-24T12:01:09.248+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-24T12:01:09.249+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
